components/Hero/Hero.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Button } from "../Button/Button";
+import Image from "next/image";
+
+export interface HeroProps extends React.HTMLProps<HTMLElement> {
+  title: string;
+  subTitle: string;
+  image: {
+    src: string;
+    alt: string;
+  };
+}
+
+export const Hero = ({ title, subTitle, image }: HeroProps) => {
+  return (
+    <section className="relative py-10">
+      <div className="flex max-w-screen-xl flex-col lg:ml-auto lg:flex-row lg:items-center">
```

Comment:
At the moment the image is being displayed at full size, you will be able to see it if you scroll horizontally. To avoid this, I think an option would be to add `overflow-hidden` to the parent element of the image.

Also, we could remove the parent container (`section`), and the `div` that contains the h1 and p tags replace it by a `section`.
Here you will find more information about [`<section> usage`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#usage_notes)

Reply:
> Also, we could remove the parent container (`section`), and the `div` that contains the h1 and p tags replace it by a `section`.
> Here you will find more information about [`<section> usage`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#usage_notes)

`section` should have the full width, and div used to center the content inside. This is why I don't remove it here. That is pretty standard practice.
I also replaced some tags with semantic alternatives, so everything should be fine, I guess.

Reply:
Take a look @LPRegen

Reply:
Regarding this commit (which is outdated), I was suggesting removing the `section` (line 16), and the `div` that contains the `h1/p` replace them with a `section`. By doing that, you will avoid having an extra wrapper (the actual `section`).
To clarify, what I am proposing is the following structure:
```tsx
<div>
  <section>
    <h1>
    <p>
  </section>
  ...
</div>
```